app/views/users/show.html.erb

```diff
@@ -1,2 +1,29 @@
-<h1>This is the method show Users!</h1>
+  <section class="container margin_10px padding_10px">
+    <div class="border container_photo">
+      <img src='<%= @user.photo %>' alt='User photo' class="user_photo">
+    </div>
 
+    <div class="border user_data_container">
+      <h2 class="user_name"><%= @user.name %></h2>
+      <h2 class="post_counter">Number of posts: <%= @user.posts_counter %></h2>
+    </div>
+  </section>
+
+  <section class="border general_container margin_10px padding_10px">
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </section>
+
+  <% @user.posts.each_with_index do |post, index| %>
+    <%= link_to user_posts_path(post.id) do %> 
+      <div class="border general_container margin_10px padding_10px group"> 
+        <h3>Post: #<%= index + 1 %></h3>
+        <p><%= post.text %></p>
+        <div class=" display_flex">
+          <p>Likes: <%= post.likes_counter %> Comments: <%= post.comments_counter %></p>
+        </div>
+      </div>
+    <% end %>
+  <% end %>
+
+  <button class="button_post margin_10px">See all posts</button>
```

Comment:
- [ ] Kindly make the `See all posts` button open the user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png).
[Hint] You can use the [link_to](https://www.rubyguides.com/2019/05/rails-link_to-method/) method to do it.

Reply:
@Omar-Muhamad 
Hello! Thank you very much for your recommendations, I am finishing applying them.